server/db/index.js

```diff
@@ -0,0 +1,17 @@
+const db = require('./conn');
+const { Sequelize } = db;
+
+//Models
+const Product = require('./Product');
+const LineItem = require('./LineItem');
+const Order = require('./Order');
+
+//Associations
+LineItem.belongsTo(Product);
+LineItem.belongsTo(Order);
```

Comment:
You will probably need Order.hasMany(LineItem).

Because if we ever want to do include lineitem model in order, we'll need this relationship. And we probably will, since every time we select an order, we can easily get the line items associated with it.